### PR TITLE
[101X][L1T][DQMOffline] Added PFMetNoMu to L1T Offline DQM

### PR DESCRIPTION
**Pull request by kreczko**
Backport of #23476

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

**Comment by 15826717775** [comment]
imaplzh7775.com

**Comment by thomreis** [comment]
+1

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

**Comment by thomreis** [comment]
please test

**Comment by thomreis** [comment]
+1

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

**Comment by dmitrijus** [comment]
+1

**Comment by dmitrijus** [comment]
please

**Comment by dmitrijus** [comment]
please test

**Comment by fabiocos** [comment]
please test

**Comment by thomreis** [comment]
+1

**Comment by thomreis** [comment]
please test

**Comment by fabiocos** [comment]
@kreczko @thomreis was the rebase correct? I understand that the subsequent L1 DQM PRs depend on this one, am I correct?

**Comment by thomreis** [comment]
There might be another one that this one depends on. I was away and lost the overview of what was already merged so I thought about trying the tests again.

File: DQMOffline/L1Trigger/python/L1TEtSumJetOffline_cfi.py [hunk 1]
```diff
@@ -41,6 +41,11 @@
 httEfficiencyBins.extend(list(xrange(400, 500, 50)))
 httEfficiencyBins.extend(list(xrange(500, 601, 10)))
 
+from L1Trigger.L1TNtuples.L1TPFMetNoMuProducer_cfi import l1tPFMetNoMu
+
+l1tPFMetNoMuForDQM = l1tPFMetNoMu.clone(
+    pfMETCollection=cms.InputTag('pfMETT1'),
+    muonCollection=cms.InputTag('muons'),
```
[the remark this answers is not in the document]
Comment (kreczko) [reply DQMOffline/L1Trigger/python/L1TEtSumJetOffline_cfi.py]: Looks like a missing bracket.

**Comment by thomreis** [comment]
please test

**Comment by andrius-k** [comment]
+1

**Comment by thomreis** [comment]
Hi @kreczko 
since the T0 has moved to the 10_2 release now this PR can be closed.

**Pull request closed without merging**